src/browser/settings.jsx

```diff
@@ -28,14 +28,21 @@ var SettingsPage = React.createClass({
     } catch (e) {
       config = settings.loadDefault();
     }
+
+    this.setState({
```

Comment:
Should be `config.showAddTeamForm = false;`. As of warning in the console log, `setState()` can't be used in `getInitialState()` of React.js.